app/mailers/settings_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class SettingsMailer < ApplicationMailer
+  default from: "derek.covey@gmail.com"
+
+  def update_email(user)
```

Comment:
This could have just as easily been placed in the UserMailer instead of creating a SettingsMailer. Works just as well this way though!